src/scss/_objects.modules.scss

```diff
@@ -90,6 +90,18 @@
   }
 }
 
+.accordion-cta-image {
+  width: 270px;
+  height: 180px;
+  object-fit: cover;
+}
+
+.resource-image {
+  width: 100px;
+  height: 100px;
+  object-fit: cover;
```

Comment:
Please add this comment to the code so that anyone who reads the code later on will know why it's there.